app/views/tag/show.html.erb

```diff
@@ -19,7 +19,10 @@
         <p><%= @wiki.latest.render_body.html_safe.match(/<p>(.+)<\/p>/).to_a.try(:first).to_s.html_safe %></p>
         <p><%= link_to("Read More &raquo;", "/wiki/#{params[:id]}") if @wiki.latest.render_body.html_safe.match(/<p>(.+)<\/p>/).length > 1 %></p>
       <% end %>
-    </div>  
+    </div> 
+  <% else %>
+    <p>This topic has no lead wiki page.</p>
+    <a class="btn btn-primary" href="/wiki/<%= params[:id] %>"><span class="glyphicon glyphicon-plus"></span> Add one now</a>
```

Comment:
oh this looks good, but do you think you could make this a fontawesome icon, as we've standardized on those:

`<i class="fa fa-plus"></i> `

I think `fa-plus` is it, but maybe check: https://fontawesome.io

Thanks!